debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerStreamingChangeEventSource.java

```diff
@@ -136,6 +137,7 @@ public void execute(ChangeEventSourceContext context) throws InterruptedExceptio
                 try {
                     connection.getChangesForTables(tablesSlot.get(), fromLsn, currentMaxLsn, resultSets -> {
 
+                        int operationOrderInInitialtx = 1;
```

Comment:
Nice! I like it. There is no need to extract `command_id` since we can compute a substitute on our own.